src/lib/make-request.ts

```diff
@@ -0,0 +1,23 @@
+import { loadBundle } from "edgespec"
+import type { Middleware } from "edgespec/middleware"
+
+import { createDatabase, type Database } from "./database/index.js"
+
+export const createMakeRequest = async (database?: Database) => {
+  const bundle = await loadBundle("./bundled-routes.js")
```

Comment:
nice...so this avoid the chicken-and-egg problem of having to uild before running the server fixture?

Reply:
yep, we can just import at runtime 👍 
although I guess this doesn't work for wintercg envs so maybe we should change this

Reply:
turned out a lot simpler than I was thinking

2e23274